Asterix/Optical_System_functions.py

```diff
@@ -846,6 +900,8 @@ def __init__(self, modelconfig, coroconfig):
             self.lyot_pup = pupil(modelconfig,
                                   prad=self.lyotrad,
                                   PupType=coroconfig["filename_instr_lyot"])
+        # this is very weird. if I put self.lyotrad it seems to broke the testbed ?
```

Comment:
euh, là, il est écrit self.lyotrad. Cela veut dire que le code ne fonctionne pas ?

Reply:
comment need to be removed